containers/scripts/init-db.sh

```diff
@@ -83,19 +70,9 @@ restore() {
     gzip -cd "$FILENAME" | podman exec -i db psql -h localhost -U openscanhub
     podman start osh-hub
     wait_for_container 'HUB'
-    # HACK: this should be turned into a function
-    # ref: https://stackoverflow.com/a/16853755/9814181
-    podman exec -i osh-hub python3 osh/hub/manage.py shell << EOF
-from django.contrib.auth import get_user_model
-User = get_user_model()
-User.objects.create_user('user', 'user@redhat.com', 'xxxxxx')
-u = User.objects.get(username='admin')
-u.set_password('xxxxxx')
-u.save()
-EOF
-
     # grant CREATEDB priviledges needed for tests
     podman exec db psql -h localhost -U openscanhub -c 'ALTER USER openscanhub CREATEDB;'
+    podman exec osh-hub python3 osh/hub/manage.py loaddata osh/hub/other/test_fixtures/users.json
```

Comment:
Why is this line attached to the block above?  It is not related to `# grant CREATEDB privileges needed for tests` in any way.

Reply:
add a blank would be much clearer.